package/MDAnalysis/topology/TOPParser.py

```diff
@@ -426,14 +427,18 @@ def parse_elements(self, num_per_record, numlines):
 
         Note
         ----
-        If the record contains atomic numbers <= 0, these will be treated as
-        dummy elements and an attempt will be made to guess the element based
-        on atom type. See issue #2306 for more details.
+        If the record contains unknown atomic numbers (e.g. <= 0), these will
+        be treated as unknown elements and assigned an empty string value. See
+        issues #2306 and #2651 for more details.
+
+        .. versionchanged X.0.0
```

Comment:
```suggestion
        .. versionchanged:: 2.0.0
```

Reply:
Oops my bad, thanks for for the review :)